server/routes/index.ts

```diff
@@ -45,6 +45,8 @@ export default function routes(router: Router, services: Services): Router {
   get('/referrals/start', (req, res) => referralsController.startReferral(req, res))
   post('/referrals/start', (req, res) => referralsController.createReferral(req, res))
   get('/referrals/:id/form', (req, res) => referralsController.viewReferralForm(req, res))
+  get('/referrals/:id/service-user-details', (req, res) => referralsController.viewServiceUserDetails(req, res))
```

Comment:
Can we add this page to our referral integration test?

Reply:
Have done this - i wasn't sure what level of detail we wanted to go into here, am testing all the service user details page, let me know if you have a better suggestion.

Reply:
Yep, I think it looks good, I'm not too sure about level of detail either when it comes to this sort of test!